code/modules/atmospherics/environmental/LINDA_turf_tile.dm

```diff
@@ -347,15 +354,35 @@
 		T.excited = FALSE
 		T.excited_group = null
 		SSair.active_turfs -= T
+		#ifdef VISUALIZE_ACTIVE_TURFS //Use this when you want details about how the turfs are moving, display_all_groups should work for normal operation
+		T.remove_atom_colour(TEMPORARY_COLOUR_PRIORITY, "#00ff00")
+		#endif
 	garbage_collect()
 
 /datum/excited_group/proc/garbage_collect()
+	if(should_display || SSair.display_all_groups)
+		hide_turfs()
 	for(var/t in turf_list)
 		var/turf/open/T = t
 		T.excited_group = null
 	turf_list.Cut()
 	SSair.excited_groups -= src
 
+/datum/excited_group/proc/display_turfs()
+	if(display_id == 0) //Hasn't been shown before
+		wrapping_id = wrapping_id % GLOB.contrast_colors.len
+		wrapping_id++ //We do this after because lists index at 1
+		display_id = wrapping_id
+	for(var/thing in turf_list)
+		var/turf/display = thing
+		display.add_atom_colour(GLOB.contrast_colors[display_id], ADMIN_COLOUR_PRIORITY)
```

Comment:
Thought the idea here is for this to be usable live ? Splashing color everywhere seems wrong. Would be nice if this was user only.

Reply:
From one side, yes but this need pass user to here. Maybe this can easily done with thing that visible only by ghosts, or admins if possible.

Reply:
My main priority was to keep lag/cpu usage down. I'm also semi unsure how I would go about hiding the tiles from normal users and not say observers, aside from invisible on some giant plane, which feels hacky

Reply:
First i think about realisation like in fake_flood halucination
<details>
https://github.com/tgstation/tgstation/blob/844c86ce93d600210e7691e2eaccc9ea42930dd5/code/modules/flufftext/Hallucination.dm#L156-L229
</details>
But it push images in client.images.
So i think that make things that invincible like ghosts and place it on turfs easier,
than hound for users in what view need pust this images.